mmdeploy/codebase/mmaction/deploy/mmaction.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+import mmengine
+from mmengine.registry import Registry
+
+from mmdeploy.codebase.base import CODEBASE, BaseTask, MMCodebase
+from mmdeploy.utils import Codebase, get_task_type
+
+
+def __build_mmaction_task(model_cfg: mmengine.Config,
```

Comment:
We do not need to define `build_func` and `build_task_processor` anymore, the default behavior should be enough. https://github.com/open-mmlab/mmdeploy/blob/01df218da54a1197929facd72875cc0cddaa6c27/mmdeploy/codebase/mmdet/deploy/object_detection.py#L16